react/tic-tac-toe/src/app/components/AuthRoute/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+import { push } from 'connected-react-router';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { LOGIN } from '../../../constants/routes';
+import { TOKEN } from '../../../constants/auth';
+
+class AuthRoute extends Component {
+  componentDidMount() {
+    if (this.props.isPrivate) {
```

Comment:
What about public screens? Should I render the login screen if im already logged?